src/ray/core_worker/core_worker.cc

```diff
@@ -1067,7 +1189,7 @@ void CoreWorker::InternalHeartbeat() {
     absl::MutexLock lock(&mutex_);
     while (!to_resubmit_.empty() &&
            current_time_ms() > to_resubmit_.top().execution_time_ms) {
-      tasks_to_resubmit.push_back(std::move(to_resubmit_.top()));
+      tasks_to_resubmit.emplace_back(std::move(to_resubmit_.top()));
```

Comment:
i actually just looked at this while running clang-tidy in core worker folder, and this isn't actually moving at all because top always returns const ref for priority queue, because moving out of priority queue causes problems for sorting but I think there should actually be a way to move it

Reply:
> because top always returns const ref for priority queue

Makes sense, updated.